### ISSUE TITLE
[feature request] file changes log

### ISSUE THREAD

**Issue by rapenne-s** [Peergos/Peergos#1184]
It would be practical to have a journal in Peergos that list the files added/modified/deleted and the timestamp of each operation.

I don't know if it's doable.

**Comment by ianopolous** [comment]
Do you mean client side (i.e. in your Peergos space encrypted)? The server doesn't have access to this information.

**Comment by rapenne-s** [comment]
Yes client side, as an user I'd like to know if I made a mistake or to remind me which actions I did recently for instance, a log is really useful for this. Seafile has such log on the sync client.

**Comment by ianopolous** [comment]
We're hoping to add an optional version history per file. Would that suffice, or do you want a more general log including things like deletion/rename?

**Comment by ianopolous** [comment]
I guess the next question is would a sync client remove your need for this?

**Comment by rapenne-s** [comment]
No, I'd like to know what happened due to the sync client anyway.

For example, when editing a file that is in the sync directory, like a libreoffice document, this can make a mess (from experience) and having the logs is important.

In fact, I can't trust a sync client to keep my data safe if it can't tell me what it did.
